test-lib.sh

```diff
@@ -1069,4 +1069,33 @@ ct_test_app_dockerfile() {
   return $ret
 }
 
+# ct_test_distgen_generated_files
+# -----------------------------
+# Argument: version which is checked
+# Result: 0 if generated files are generated properly
+#         1 if generated files are not the same as in PR.
+ct_test_distgen_generated_files() {
+  # we need to switch to root directory.
+  # The tests are called from 'test' directory
+  pushd "../"
+  if [[ ! -f "./manifest.sh" ]]; then
+    echo "This container does not use dist-gen. Skipping it."
+    popd
+    return 0
+  fi
+
+  echo "Generating all files by 'make generate-all'"
+  make generate-all
+  # We do not want to check container-common-script directory
+  if git diff --exit-code -- ':!common'; then
+    echo "Pull request does not contain properly generated files."
+    echo "Please run these command in order to update pull request:"
+    echo "'make generate-clean'"
```

Comment:
Are you sure about `generate-clean` we don't have it in Python container.

Reply:
Updated with 'make clean-version', In my mind, I have 'make generate-clean' for some reason.